docs/src/components/install/jb/binary.mdx

```diff
@@ -2,16 +2,19 @@ The `jb` binary is primarily distributed using [GitHub releases](https://github.
 
 #### Linux, macOS
 
-For Linux and macOS, download the binary for your architecture and put it somewhere on your `$PATH`:
+For Linux and macOS, download the binary for your architecture, put it somewhere on your `$PATH`, and make it an executable:
 
 ```bash
 # Linux amd64:
 sudo curl -Lo /usr/local/bin/jb https://github.com/jsonnet-bundler/jsonnet-bundler/releases/latest/download/jb-linux-amd64
+sudo chmod +x /usr/local/bin/jb
```

Comment:
For consistency with the release notes (https://github.com/grafana/tanka/blob/main/.drone/release-note.md)
Also, since this is called with sudo, this would mean that only root could execute it, right? `a+x` fixes that

```suggestion
sudo chmod a+x /usr/local/bin/jb
```

Reply:
To my knowledge if you don't specify the owner, group, or others flag then the permission will apply to all. In other words, `+x` and `a+x` are equivalent.
However, happy to change it for consistency.